receiver/jaegerreceiver/factory.go

```diff
@@ -128,11 +128,23 @@ func createDefaultConfig() configmodels.Receiver {
 			ThriftHTTP: &confighttp.HTTPServerSettings{
 				Endpoint: defaultHTTPBindEndpoint,
 			},
-			ThriftBinary: &confignet.TCPAddr{
+			ThriftBinary: &ProtocolUDP{
 				Endpoint: defaultThriftBinaryBindEndpoint,
+				ServerConfigUDP: ServerConfigUDP{
+					QueueSize:        defaultAgentQueueSize,
+					MaxPacketSize:    defaultAgentMaxPacketSize,
+					Workers:          defaultAgentServerWorkers,
+					SocketBufferSize: 0,
```

Comment:
Why is this set to zero and not the default?

Reply:
Let's make default constant for 0